docs/apache-airflow-providers-amazon/connections/aws.rst

```diff
@@ -60,10 +60,10 @@ automatically the credentials from there.
 .. caution:: If you do not run "airflow connections create-default-connections" command,
     most probably you do not have ``aws_default``. For historical reasons, the Amazon Provider
     components (Hooks, Operators, Sensors, etc.) fallback to the default boto3 credentials strategy
-    in case of a missing Connection ID. This behaviour is deprecated and will be removed in a future releases.
+    in case of a missing Connection ID.
 
     If you need to use the default boto3 credential strategy (credentials in environment variables, IAM Profile, etc.)
-    please provide ``None`` instead of a connection ID.
+    without get warning in your logs please provide ``None`` instead of a missing connection ID.
```

Comment:
Some grammar suggestions, let me know if I'm interpreting this correctly:
```suggestion
    please provide ``None``, instead of a missing connection ID, to avoid getting a warning in your logs.
```